proposals/Level-1.md

```diff
@@ -1,4 +1,4 @@
-# Level 1 exception handling
+ Level 1 exception handling
```

Comment:
nit: I'd leave the `#` so this is a heading.

Reply:
Done.